std_msgs/Header header
#geometry_msgs/Header trans_header

# Integer class IDs for each bounding box
geometry_msgs/Vector3[] transition
geometry_msgs/Quaternion[] rotation

# String class IDs for each bouding box
string[] class_names


